setup.cfg

```diff
@@ -49,6 +49,8 @@ install_requires =
     rasterio>=1.0.20,<2
     # rtree 1+ required for len(index), index & index, index | index
     rtree>=1,<2
+    # scikit-image required for find_contours
+    scikit-image>=0.15.0,<0.20
```

Comment:
This definitely shouldn't be required if it's optionally needed by a single dataset. Let's move it to the datasets extra, just like pycocotools and rarfile.